### ISSUE TITLE
Rule `aws_guardduty_member_invalid_email` has bad regex

### ISSUE THREAD

**Issue by acwwat** [terraform-linters/tflint-ruleset-aws#923]
A [recent update](https://github.com/terraform-linters/tflint-ruleset-aws/commit/862736e9f3682f131dfdca241956c824c83df266 ) has introduced a badly generated regex pattern to the rule [`aws_guardduty_member_invalid_email`](https://github.com/terraform-linters/tflint-ruleset-aws/blame/master/rules/models/aws_guardduty_member_invalid_email.go) which causes the check to always fail:

```go
		pattern:       regexp.MustCompile(`^See rules in parameter description$`),
```

The only workaround is to disable the rule for now. Please address the issue upstream at the source where the code is generated so that the rule can be updated with the correct email regex.

**Comment by bendrucker** [comment]
> rule can be updated with the correct email regex.

There's no such thing, there's just this fake pattern AWS has added, and the "nothing" that was there before. The fix is just to detect and ignore it. This rule just checks the length of the input, you are not missing much by disabling.
